src/unix/async.c

```diff
@@ -231,6 +231,18 @@ int uv__async_start(uv_loop_t* loop, struct uv__async* wa, uv__async_cb cb) {
 }
 
 
+int uv__async_fork(uv_loop_t* loop, struct uv__async* wa) {
+  if (wa->io_watcher.fd == -1) /* never started */
+    return 0;
+
+  uv__async_stop(loop, wa);
+
+  uv__async_init(wa);
+
+  return uv__async_start(loop, &loop->async_watcher, uv__async_event);
+}
```

Comment:
Oh man, I was really hoping this would get merged before #1248. Rebasing is not fun :/ 

I'll see what I can do.